challenges/backend/package.json

```diff
@@ -5,17 +5,29 @@
   "main": "dist/index.js",
   "scripts": {
     "start": "npx ts-node src/app/main.ts",
-    "test": "./node_modules/.bin/mocha --require ts-node/register ./src/**/*.spec.ts"
+    "test": "./node_modules/.bin/mocha --require ts-node/register ./src/**/**/*.spec.ts",
+    "test:cov": "nyc --reporter=html --reporter=text ./node_modules/.bin/mocha --require ts-node/register ./src/**/**/*.spec.ts"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
```

Comment:
Would be nice to clean up the dependencies:
@types/inversify, @types/node, ts-node, and typescript are examples of devDependencies.